bot.py

```diff
@@ -39,6 +39,11 @@
 except KeyError:
     REDIS_HOST = "localhost"
 
+try:
+    MODERATORS = os.environ["MENSTRUATION_MODERATORS"]
+    MODERATORS = list(MODERATORS.split(","))
```

Comment:
Please do this in one expression. Constants shouldn't be reassigned. I propose sth like

`MODERATORS = (os.environ["..."] or "").split(",")`